plugin/src/main/scala/com/lightbend/paradox/sbt/ParadoxPlugin.scala

```diff
@@ -64,7 +71,7 @@ object ParadoxPlugin extends AutoPlugin {
     paradoxProperties ++= linkProperties(scalaVersion.value, apiURL.value, scmInfo.value),
 
     sourceDirectory in paradox := sourceDirectory.value / "paradox",
-    sourceDirectories in paradox := Seq((sourceDirectory in paradox).value),
+    sourceDirectories in paradox := Seq((sourceDirectory in paradox).value) ++ paradoxOverlayDirectories.?.value.getOrElse(Nil),
```

Comment:
Instead of doing `++ paradoxOverlayDirectories.?.value.getOrElse(Nil)`, `paradoxOverlayDirectories` should be initialized to `Nil`.  Otherwise the following blows up:

```scala
paradoxOverlayDirectories in DocsFirst += baseDirectory.value / "src" / "commonFirst"
```